src/main/java/org/hl7/davinci/priorauth/ProcessClaimItemTask.java

```diff
@@ -142,4 +173,39 @@ private boolean process() {
 
         return ret;
     }
+
+    private Disposition sendAndGetDisposition(Bundle bundle, int seq, String address) throws IOException {
+
+        // create post request
+
+        OkHttpClient client = new OkHttpClient();
+        RequestBody body = RequestBody.create(null, bundle.toString());
+        Request request = new Request.Builder().url(address).post(body).build();
+        // if no response in alloted time return some default value
+        try (Response response = client.newCall(request).execute()) {
+            String answer = response.body().string();
+            return convertStringTDisposition(answer);
+        } catch (Exception e) {
+            return Disposition.CANCELLED;
+        }
+    }
+
+    private Disposition convertStringTDisposition(String answer) {
```

Comment:
This method should be defined in the Disposition enum (`FhirUtils.java`). You should be able to copy the `fromString` method in the ReviewAction enum changing types as appropriate.

Reply:
Is this method no longer used? Looks like it was deleted

Reply:
I moved it to the enum in FHIRUtils